config/config.go

```diff
@@ -919,6 +931,13 @@ func PrintPelicanVersion(out *os.File) {
 	fmt.Fprintln(out, "Built By:", GetBuiltBy())
 }
 
+func LogPelicanVersion() {
+	log.Debugf("Version: %s", GetVersion())
```

Comment:
If all we're doing is concatenating strings together, the `Debugln` variant is more elegant.  Additionally given the information used to be printed all the time by default; let's move it to `Info`:

```suggestion
	log.Infoln("Version:", GetVersion())
```

(and the subsequent lines as well.)